src/dev.html

```diff
@@ -18,6 +18,8 @@
 
     <header id="site-header"></header>
 
+    <mkt-nav role="navigation" id="site-nav"></mkt-nav>
```

Comment:
Let's inject it after. Don't want to depend on that being in the initially served HTML and it's more appropriate as we render most things with js. It's still flexible since apps can render it wherever they want. ideally I'd get rid of the cloak/lightbox stuff hanging around as well.